tests/lib/pkgdb.sh

```diff
@@ -850,3 +850,80 @@ distro_upgrade() {
             ;;
     esac
 }
+
+distro_list_packages() {
```

Comment:
Can you please spin this off into a `distro-tool` and perhaps leverage mvo's `apt-tool` as a base?

Reply:
with the new guidelines it would be invoked as suite.packages <subcommand>

Reply:
I'm not as familiar with the apt API used by apt-tool to say whether it does the same as dselect-upgrade. Perhaps @mvo5 knows more.